test/Libraries/AnalysisTests/SurfaceDataTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using Analysis;
+using Autodesk.DesignScript.Geometry;
+using NUnit.Framework;
+
+namespace AnalysisTests
+{
+    [TestFixture]
+    class SurfaceDataTests : TestServices.GeometricTestBase
+    {
+        /// <summary>
+        /// This test method will execute the BySurfaceAndPoints method from the SurfaceData class passing different parameters
+        /// </summary>
+        [Test, Category("UnitTests")]
+        public void SurfaceAnalysisDataBySurfaceAndPointsTest_Exceptions()
+        {
+            //Act
+            var sad = SurfaceData.BySurfaceAndPoints(TestSurface(),TestUvs());
+
+            //Assert
+            Assert.NotNull(sad);
```

Comment:
Unrelated, but what is the difference between `Assert.IsNotNull` and `NotNull`?

Reply:
@aparajit-pratap 
I was reading that IsNotNull and NotNull are the same but IsNotNull is compatible to previous versions of nUnit then I updated two lines for using IsNotNull.

commit: https://github.com/DynamoDS/Dynamo/pull/10989/commits/0fbc2034d4f413ca7cfe9ae76c525eacb51be946